src/sys/mman.rs

```diff
@@ -205,6 +205,11 @@ libc_bitflags! {
         /// Place the mapping at exactly the address specified in `new_address`.
         #[cfg(target_os = "linux")]
         MREMAP_FIXED;
+        /// Works in conjunction with `MREMAP_MAYMOVE` but does not unmap `old_address`.
+        /// Note that, in this case, `old_size` and `new_size` must be the same.
+        /// [`mremap(2)`]: https://man7.org/linux/man-pages/man2/mremap.2.html
```

Comment:
No need to link the man page here, it was already [there](https://docs.rs/nix/latest/nix/sys/mman/fn.mremap.html)

```suggestion
```